src/ui/AddLinkBar/AddLinkBar.jsx

```diff
@@ -0,0 +1,24 @@
+import "./AddLinkBar.css";
+import { ADD_ICON } from "./constans";
+
+export function AddLinkBar() {
+  return (
+    <div className="add-link">
+      <div className="add-link-bar">
+        <div className="add-link-icon-input">
+          <img
+            className="add-link-icon"
+            src={ADD_ICON}
+            alt="링크 추가하기 아이콘"
+          />
+          <input
+            className="add-link-input"
+            type="text"
+            placeholder="링크를 추가해 보세요"
+          ></input>
```

Comment:
P3:
정말 제 개인적인 취향이지만 저는 태그안에 내용이 없으면 아래와 같은 방식을 선호합니다.

```suggestion
          <input
            className="add-link-input"
            type="text"
            placeholder="링크를 추가해 보세요"
          />
```